exporter/stackdriver/stats_test.go

```diff
@@ -99,15 +114,15 @@ func TestExporter_makeReq(t *testing.T) {
 
 	start := time.Now()
 	end := start.Add(time.Minute)
-	count1 := view.CountData(10)
-	count2 := view.CountData(16)
-	sum1 := view.SumData(5.5)
-	sum2 := view.SumData(-11.1)
-	mean1 := view.MeanData{
+	count1 := exporter.AggregationData{Count: 10}
+	count2 := exporter.AggregationData{Count: 16}
+	sum1 := exporter.AggregationData{Mean: 5.5, Count: 1}
```

Comment:
I personally prefer the old style using the functions, it was consistent and less confusing but it didn't leak
information about the implementation, the user shouldn't have to know the relationship between Mean and Count and Sum. Could we please revert AggregationData as the functions were articulate already?

Reply:
The functions actually don't return exporter.AggregationData, they are part of the view package so they can't be used here.